### DISCUSSION TITLE
Can I put Radius effect in UI Node?

### DISCUSSION THREAD

**Discussion by dolto** [bevyengine/bevy#9271] [Q&A] [answered]
Hello, I'm practicing to get used to writing UI in BEVY.

First, I design it in figma, and then I practice it in a way that implements the ui in bevy.

Currently, while designing in figma, I put the radius effect in ui, but it is hard to find the function in bevy.

Even if I search on Google, it's hard to find the relevant results.

I saw a similar thread in 2022, but I couldn't understand it properly because my English was not good.

Is there a way?
Or is there no attribute for putting radius in bevy yet?

![image](https://github.com/bevyengine/bevy/assets/43398206/5634c6ef-642a-4f6e-83cf-a11fc8a1cd00)

**Comment by nicopap** [comment, marked as answer]
Currently bevy doesn't support border radius.

But it plans to add it in the future. The feature exist, just not merged yet. It's probably going to be available in 0.12.

* You can track progress here: https://github.com/bevyengine/bevy/pull/8973
